tests/phpunit/Mutation/NodeTraverserFactoryTest.php

```diff
@@ -35,70 +35,20 @@
 
 namespace Infection\Tests\Mutation;
 
-use function array_map;
 use Infection\Mutation\NodeTraverserFactory;
-use Infection\Visitor\FullyQualifiedClassNameVisitor;
-use Infection\Visitor\NotMutableIgnoreVisitor;
-use Infection\Visitor\ParentConnectorVisitor;
-use Infection\Visitor\ReflectionVisitor;
+use PhpParser\NodeTraverser;
 use PhpParser\NodeVisitorAbstract;
 use PHPUnit\Framework\TestCase;
 
 final class NodeTraverserFactoryTest extends TestCase
 {
-    /**
-     * @var NodeTraverserFactory
-     */
-    private $traverserFactory;
-
-    protected function setUp(): void
-    {
-        $this->traverserFactory = new NodeTraverserFactory();
-    }
-
     public function test_it_can_create_a_traverser(): void
     {
-        $traverser = $this->traverserFactory->create([]);
-
-        $visitors = array_map('get_class', $traverser->getVisitors());
-
-        $this->assertSame(
-            [
-                50 => NotMutableIgnoreVisitor::class,
-                40 => ParentConnectorVisitor::class,
-                30 => FullyQualifiedClassNameVisitor::class,
-                20 => ReflectionVisitor::class,
-            ],
-            $visitors
-        );
-    }
-
-    public function test_it_can_create_a_traverser_with_extra_visitors(): void
```

Comment:
I wouldn't be a big fan in general but since the node traverser does not exposes any state, how about checking the registered visitors via class reflection? That way you can also check that the additional node ignorers are also properly registered (and in the right order) to `NotMutableIgnoreVisitor`